solutions/kubernetes/cortx-deploy-functions.sh

```diff
@@ -151,7 +150,6 @@ function update_solution_config(){
         yq e -i '.solution.common.resource_allocation.zookeeper.resources.limits.cpu = "1000m"' solution.yaml
 
         yq e -i '.solution.common.resource_allocation.kafka.storage_request_size = "8Gi"' solution.yaml
-        yq e -i '.solution.common.resource_allocation.kafka.log_persistence_request_size = "8Gi"' solution.yaml
```

Comment:
@shailesh-vaidya, not able to see this removal in latest services framework? from where did we get this?

Reply:
This is added as part of https://github.com/Seagate/cortx-k8s/pull/233/